enchanted-swiftie/cogs/songs.py

```diff
@@ -0,0 +1,66 @@
+import billboard
+import datetime as dt
+from discord.ext import commands
+
+
+class Songs(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.singles_dict = {}
+        self.last_checked = None
+
+    @commands.command(aliases=["Singles"])
+    async def singles(self, ctx):
+        """Returns all of Taylor's singles that peaked in the hot 100"""
+        # Goes over all the charts up to current date
+        while self.last_checked + dt.timedelta(7) < dt.datetime.date(dt.datetime.now()):
```

Comment:
`dt.timedelta(7)` is not clear, 7 what?
when positional arguments are not clear, use keyword arguments: `dt.timedelta(days=7)`
or even clearer: `dt.timedelta(weeks=1)`